doc/query.md

```diff
@@ -87,9 +158,14 @@ use these field names.
 Indexing of array fields is supported. See "Array fields" below for the indexing and
 querying semantics.
 
+#### Restrictions: 
```

Comment:
Moved to the creating section and colon is removed.  Done in ee198020c5b7e28d8fcf044f68f77f22fa1db7e2.